src/app/services/api-data.service.ts

```diff
@@ -101,6 +145,29 @@ export class ApiDataService {
     );
   }
 
+  getAllSpecimensForProject(project: string) {
+    const url = this.hostSetting.host + 'specimen/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```

Comment:
same thing here, headers is not required.

Reply:
done in c257a2a